src/main/java/pl/wedding/management/service/weddinghall/model/TypesOfHalls.java

```diff
@@ -0,0 +1,18 @@
+package pl.wedding.management.service.weddinghall.model;
+
+public enum TypesOfHalls {
+    CARNIVAL_HALL,
```

Comment:
IMHO those suffixes are not required. It's TypesOfHall enum, it's obvious that it's _HALL...